### ISSUE TITLE
Bookmarking variants

### ISSUE THREAD

**Issue by AlistairNWard** [iobio/gene.iobio.vue#132]
If I select a gene from the left list, it is permanently highlighted in the proband card. It isn't clear to the user why. I am just using the left menu to identify the variant, not necessarily to mark it as a bookmarked. There also appears to be no way to remove the flag, so this variant remains highlighted, even though I never wanted to highlight it in the first place!

Do we want to add check boxes or dismiss variant options in the left panel? If I look through the variants in this list, I will likely deem some to be uninteresting, and it would be nice to be able to remove them from the list. Additionally, marking others as interesting, adding a star or something, would be useful.

**Comment by AlistairNWard** [comment]
I guess there were three things, so maybe this should be broken up. But open SSC sample 13529.p1 from hub in gene ATP7B.

A. Hightlighting

1. Click on the variant at the top of the left list.
2. Click on the 2nd variant from the left in the ranked variants table.

![Screen Shot 2018-07-18 at 9.10.33 AM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5750332babe50b1300c53bf6/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77246455fb6b7191a371f1132b4f5c5c5edd3c4a4356bcbbd671f3e57862804e0ebc836e0b26926c47aebc821ee901ba228e4cd7e2b9de1ad2f42f67b6d5db1063474f4dcbed5727a19e41c95df54ae84c5ca0173e65e1702431.png)

There is different highlighting in the ranked variants table, but there is also a square around the variant in the proband card, and a circle around the selected variant. There is no way for me to get rid of the highlighting in the proband card.

B. Bookmarking

We don't need to call it bookmarking, or have it work how it used to in original gene, but we do need to be able to mark some variants as interesting for later viewing.

C. Variant removal

Kind of the opposite of bookmarking. We need to be able to trim down the variants in the left panel. I can imagine that as we develop clin, we will be able to refine the variants that appear in the left list, e.g. all de novos, etc. As we review the variants, we would need to be able to remove variants from the list, ultimately leaving us with the variants that we would propagate to the final report.

**Comment by tonydisera** [comment]
Hi Al.  I just made some fixes so that the 'flagged' variants are no longer highlighted with the light blue border.  Instead, the Ranked Variants table has a row 'Flagged'.  A red flag indicates that the variant passed a filter; a blue flag indicates the user flagged a variant.  


![Screen Shot 2018-07-18 at 2.51.56 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5759f0b40b37b00e00e7d3d4/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77246455fb6b7191a37af1172a4f5a595ecc3c4a4356bcbbd671f3e57862804e0ebc836e0b26926c47aebc821ee901ba228e4cd7e2b9de1ad2f42f67b6d5db1063474f4dcbed5727a19e41c95df54ae84952a5163560e17f2431.png)

This brings up a point of confusion.  Right now, the 'Flag variant' button only appears for variants that are not already flagged.   This may need to be reworked once we figure out how we want the user to be able to annotate/edit the list of flagged variants.  (I moved your "C" item to another issue .)
![Screen Shot 2018-07-18 at 2.52.52 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5759f0b40b37b00e00e7d3d4/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77246455fb6b7191a37af117294f5a5d5ecc3c4a4356bcbbd671f3e57862804e0ebc836e0b26926c47aebc821ee901ba228e4cd7e2b9de1ad2f42f67b6d5db1063474f4dcbed5727a19e41c95df54ae84952a6113567ef702431.png)
